docs_2.0/manual_EN/4.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -70,12 +85,80 @@ Search for vertices with properties matching a certain pattern:
     Got 2 rows (time spent 2407/3244 us)
     ```
 
-## MATCH on edge types
+* Example 3: Search for the player whose name is "Tim", return the player's relationships (edges) with others, as well as the names of these people. The `r` is a user-defined variable that represents an edge. `type(e)` to refer to the edge type.
```

Comment:
no need to explain [e](Line91) or [e:serve](Line108)?

Reply:
Added. Thanks!